### PR TITLE
navbar collapses to hamburger-menu

### LINKED ISSUE #86: Turn main navi into hamburger menu for narrow windows
We should not always display the main navi as a vertical list for narrow windows (e.g. mobiles). Instead, we should hide the menu and only display it on click on a hamburger icon or so (e.g. like http://www.201-created.com does it).

I think Bootstrap has built-in mechanisms for achieving that.

### PR DESCRIPTION
**Pull request by trabaldini**
closes #86

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -2,23 +2,28 @@
...
-          </li>
-          {% for link in site.navigation %}
-            {% assign active = false %}
-            {% if page.url contains link.url %}
-              {% assign active = 'active' %}
-            {% endif %}
-            <li class="page">
-              <a href="{{link.url}}" class="{{active}}"><span>{{link.title}}</span></a>
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#collapsingNavbar">
+          ☰
```
Comment (marcoow) [inline _includes/nav.html]: I think we should use the respective character from font-awesome here instead.

**Review by marcoow — COMMENTED** [review]
I think this should also be possible with CSS only which would be preferrable so we don't have to introduce additional JS dependencies.

See https://siongui.github.io/2017/03/23/css-only-bootstrap-responsive-navbar/

**Review by marcoow — COMMENTED** [review]
Hm, this doesn't work for me - nothing happens when I click on the hamburger.

**Comment by trabaldini** [comment]
Ok strange, on my localhost it's working. The simplabs-icon disappears and I can open the menu and the pages in it

**Comment by marcoow** [comment]
Maybe you haven't committed some CSS changes? The changes in this PR seems pretty small. Also we should use `cursor: pointer;` for the hamburger menu.

**Comment by marcoow** [comment]
Ah, sorry - didn't have the latest code actually. The hamburger menu seems a bit big though…

**Pull request merged**